plugins/importer/src/main/java/com/managemc/importer/command/handler/CmdImportHandler.java

```diff
@@ -61,6 +68,10 @@ public void processCommandAsync() {
       case MAX_BANS_PLUS:
         importerService.importMaxBans(sender);
         return;
+      case ESSENTIALS_X:
+        File dataFolder = bukkitWrapper.getOtherPlugin("Essentials").getDataFolder();
```

Comment:
we have a wrapper class for common Bukkit methods so that they can be stubbed in our test suite